typings/guppy-defs-1759992a3885cb99d3ead04241f318a074c9f963.d.ts

```diff
@@ -1,5 +1,9 @@
 /* eslint-disable camelcase */
-// Code - third_party/static/guppy-b5055b/src/guppy.js
+// Code - third_party/static/guppy-175999/src/guppy.js
+// This guppy version corresponds to the commit on June 15 2020. 
+// For reference, visit:
+// eslint-disable-next-line max-len
```

Comment:
I don't think you need to worry about max-len for urls, right? eslint has an option that doesn't apply the check when the line is a URL. /cc @Hudda 

(Btw @Hudda this keeps coming up. Could you confirm that URLs are ok? If so, would it be possible to update the coding style guide on the wiki accordingly so that folks will know about this exception? Thanks!)

Reply:
Done -- the lint checks passed with no errors when I removed the `eslint-disable`.

Reply:
@seanlip Line containing URLs is allowed to have length of more than 80 characters. I have added this rule to the style guide.

Reply:
Thanks @Hudda!